modules/backend/widgets/form/assets/js/october.form.js

```diff
@@ -220,6 +221,54 @@
             .nextUntil('.section-field').hide()
     }
 
+    /*
+     * Bind validation checks on available fields, triggered when
+     * the fields is blurred or changed
+     */
+    FormWidget.prototype.bindValidation = function () {
+        let fields = this.getFieldElements(),
+            form = this.$form;
```

Comment:
When a variable contains a jQuery reference it's common to prefix it with `$` to indicate it's a jQuery handled element instead of just a plain JS element.